RadixWallet/Core/SargonExtensions/IntoSargon+Bridge.swift

```diff
@@ -166,8 +166,8 @@ extension ThirdPartyDeposits {
 }
 
 extension OrderedSet<EntityFlag> {
-	func intoSargon() -> [Sargon.EntityFlag] {
-		map { $0.intoSargon() }
+	func intoSargon() -> Sargon.EntityFlags {
```

Comment:
N.B. not really used... only used to be able to used `AccountOrAddressOf` enum and pass `Account` to Sargon, for AssetsTransfers, but the entity flags are not being read/ used by that....